.github/workflows/go-cross.yml

```
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        go-version: [ 1.19, '1.20' ]
+        go-version: [ 1.18, 1.19, 1.20 ]
```

```suggestion
        go-version: [ 1.19, '1.20' ]
```

the quotes are required